btr-web/btr-common-components/plugins/vue-country-flag.ts

```diff
@@ -0,0 +1,5 @@
+import CountryFlag from 'vue-country-flag-next'
```

Comment:
Did we really needed it as global component, could we not just load it when needed ?

Doesnt need changing, just wondering about reasoning.

Reply:
Also, were you not able to use existing component BcrosCountryFlag

Reply:
We already had vue-country-flag as an import I mostly looked to copy what BoR was doing